README.md

```diff
@@ -4,11 +4,20 @@
 
 This connector allows running sql on TileDB arrays via Presto.
 
+TileDB is a library and format that efficiently manages large-scale,
+n-dimensional, dense and sparse array data. For more information about TileDB
+see the [official TileDB Documentation](https://docs.tiledb.io/en/latest/introduction.html)
+
 ## Usage
 
 The TileDB presto connector supports most sql operations from prestodb. Arrays
 can be references dynamically and are not required to be "pre-registered"
-with presto. To query a tiledb array simply reference the URI in the from
+with presto. No external service, such as hive is required either.
+ 
+Predicates pushdown is supported for dimension fields.
```

Comment:
Predicate pushdown is supported for dimension fields (range-queries) and column / attribute selection ( projection-queries).

Reply:
Predicate pushdown for the tiledb connector doesn't support attributes, we only filter dimensions so we can use a subarray. Attributes we let presto filter.